tunnel/server/json.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 SEQSENSE, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"encoding/json"
+	"reflect"
+	"strings"
+	"unicode"
+)
+
+type response struct {
+	value interface{}
+}
+
+func (j *response) MarshalJSON() ([]byte, error) {
```

Comment:
Why this func is needed?
Is just using `json.Marshal` insufficient?

Reply:
Ah, it's for making keys lower camel case, right?

Reply:
That's right